content/03.aim1.md

```diff
@@ -62,4 +62,56 @@ NMI is an normalization of the Mutual Information (MI, measures the dependence o
 
 ![](images/nmi.png)
 
-Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters. 
+Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
+
+#### VAE performance on real single cell datasets
+#### 1. real single cell datasets:
+ To evaluation the performance of VAE and the other dimension reduction approaches, we select 8 real single cell datasets with true cell type labels.
+
+| dataset | ncells | ngenes | tissue | organism | celltypes |
+| ------- | ------ | ------ | ------ | -------- | --------- |
+| yan | 90 | 20214 | Embryo Devel | human | 6 |
+| goolam | 124 | 41428 | Embryo Devel | mouse | 5 |
+| pollen | 301 | 23730 | Developing Cerebral Cortex | human | 11 |
+| patel | 430 | 5948 | Glioblastoma | human | 12 |
+| nestorowa | 1656 | 4773 | Glioblastoma | human | 12 |
+| Petropoulos | 1529 | 26178 | Embryo Devel | human | 6 |
+| HCA melanoma data | 6196 | 22489 | Lymph node | Mus musculus | 9 |
+| baron-human | 8569 | 20125 | Pancreas | human | 14 |
+
+#### 2. Visualization of real single cell data based on differnt structure of VAE: VAE_depth3 (3 hidden layer), VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE, ZIFA, umap and PCA
+We tested the performance of VAE and another 4 methods (t-SNE, ZIFA, umap and PCA) on these real datasets with different number of cells, genes and cell types. Figure 4 shows the 2-D visulization of the real single cell datasets. PCA and ZIFA generally performs better when number of cells is small, but with the increase of number of cells, VAE shows a good power to differentiate different celltypes.
+
+![](images/real.data.vis.png)
+
+Figure 4: 2D Visualization of real single cell data using VAE_depth3 (3 hidden layer), VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE, ZIFA, umap and PCA.
+
+#### 3. Performance evaluation
+Evaluation of model performance is based on three different types of metrics: clustering based, average silluette score and knn-based.
+
+*Clustering based approach: k-means clustering is performed based on the latent space, the model performance is measured by NMI and ARI. 
+
+![](images/true.celltype.clustering.png)
+
+Figure 5:  Performance comparison among VAE, t-SNE, ZIFA, umap and PCA based on k-means. For dataset with small number of cells, the performance of t-SNE, umap, ZIFA and PCA is better than VAE. VAE and umap outperforms t-SNE, ZIFA and PCA whith increase number of cells. 
```

Comment:
this looks like `umap` outperforms everything else. How hard would it be to put the tybalt models through `tsne` _and_ `umap`. This may be a better comparison. `tybalt` outperforms `tnse`, but do `tybalt` features process with `umap` outperform raw features through `umap`?

Reply:
also, it is probably best to keep the columns consistent between the performance figures and the clustering figures. (i.e. the tybalt depth is switched)

Reply:
How hard would it be to put the tybalt models through tsne and umap.
It should be easy to do umap based on tybalt models, but it may take sometime to run it. I will update it through next PR. 

tybalt outperforms tnse, but do tybalt features process with umap outperform raw features through umap?

This is an interesting question, I will take a look at it.